doc/src/template.md

```diff
@@ -1 +1,66 @@
-# Challenge Template
+# Challenge XXXX[^challenge_id]: Challenge Title
```

Comment:
Is "Challenge" the right wording?

Reply:
Any suggestions? We could use `Verification Task` or something like that.